static/UserDefault.json

```diff
@@ -15,14 +15,14 @@
             "LastScore": 0,
             "LastCompletedChallenge": "00000000-0000-0000-0000-000000000000",
             "Locations": {
-                "location_ica_facility": {
+                "location_parent_ica_facility": {
```

Comment:
Applies to all key changes in this file.

I don't see existing userprofiles being converted from sublocation -> parents, this **must** be done.

Reply:
Existing user profiles already use parents. They don't use sublocation at all. That's why I made that change.

https://github.com/thepeacockproject/Peacock/blob/b3d0353df1b69740e9c7c9b2417bd686ee1e36b8/components/candle/masteryService.ts#L186

Reply:
Ah I see.